src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/SimpleFormFactory.php

```diff
@@ -77,17 +77,4 @@ protected function createListener($container, $id, $config, $userProvider)
 
         return $listenerId;
     }
-
-    protected function createEntryPoint($container, $id, $config, $defaultEntryPoint)
```

Comment:
Same function with the same body defined in a parent class, so BC is not introduced. 
In nutshell, this is duplicate code if \Symfony\Bundle\SecurityBundle\DependencyInjection\Security\Factory\FormLoginFactory::createEntryPoint